server/routes/users.ts

```diff
@@ -152,73 +100,58 @@ router.post('/users', async (req: Request, res: Response, next) => {
                         attributes: ['name'],
                     },
                 ],
-            });
-            db.Token.create({
-                userId: addedUser.id,
-                token: token,
-            });
+            }));
 
-            const [message, status] = helper.sendVerEmail(
-                username,
-                req.headers,
-                token,
-                'confirmation',
-            );
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+        addedUser && (await db.Token.create({ userId: addedUser.id, token }));
```

Comment:
I think this works too, so you can remove the eslint disable:
```suggestion
        if (addedUser) await db.Token.create({ userId: addedUser.id, token });
```

Reply:
likewise